app/views/coronavirus_landing_page/components/shared/_section.html.erb

```diff
@@ -11,6 +11,12 @@
             href: item["url"],
             text: item["label"]
           }%>
+          <% item_description = content_item_description item["url"] %>
```

Comment:
Should this be moved to the top of the file to make it a bit cleaner?

Reply:
As it's only used once, I think it's nice to have the definition close to where it was used. If it was used multiple times, then I think moving it to the top would be a good idea.

Reply:
We can't, unfortunately, move this to the top as it's within a loop and we need to generate this based on the scope

Reply:
@DilwoarH ah yes – that makes sense